tests/validators/dict_validator_test.py

```diff
@@ -563,7 +563,7 @@ class UnitTestDictValidator(DictValidator):
                 'value': DecimalValidator(),
                 'optional_value': DecimalValidator(),
             }
-            required_fields = ['name', 'value']
+            required_fields = {'name', 'value'}
```

Comment:
This looks confusing... a list but with curly braces?

Reply:
It's not a list, it's a set. :)

```
>>> type([1, 2, 3])
<class 'list'>

>>> type({1, 2, 3})
<class 'set'>
```

The reason for this change is that `DictValidator.required_fields` is actually a set and not a list (when creating a `DictValidator` with the `required_fields` parameter, it is expected as a list, but then converted to a set). So when subclassing a DictValidator and setting `required_fields`, technically it needs to be a set too.

However, thinking about this and looking at the source code, I noticed some inconsistencies/weirdnesses about this (like, when it's saved as a set, why is the construction parameter defined as a list? It could just be any Iterable actually. Also, it doesn't actually make that much sense to save it as a set...).

I think I might change some of this (now or later). But for this PR, the reason is just "otherwise the typing is wrong". ^^